### PR TITLE
Updated the description of the pull to update control

### PR DESCRIPTION
**Pull request by gregwoo-microsoft**
Better reflects that this control is touch-centered

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 1]
```diff
@@ -459,7 +459,7 @@
           "Title": "PullToRefresh",
           "Subtitle": "Provides the ability to pull on a collection of items in a list/grid to refresh the contents of the collection.",
           "ImagePath": "ms-appx:///Assets/PullToRefresh.png",
-          "Description": "PullToRefresh Provides the ability to pull on a collection of items in a list/grid to refresh the contents of the collection.",
+          "Description": "Pull-to-refresh lets a user pull down on a list of data using touch in order to retrieve more data. Pull-to-refresh is widely used on devices with a touch screen.",
```
[the remark this answers is not in the document]
Comment (gregwoo-microsoft) [reply XamlControlsGallery/DataModel/ControlInfoData.json:462]: I copied the description from the docs: https://docs.microsoft.com/en-us/windows/apps/design/controls/pull-to-refresh#:~:text=%20Implement%20pull-to-refresh%20%201%20Wrap%20your%20list,%28for%20example%2C%20from%20a%20button%20click%29.%20More%20

It seems like the original control name was "Pull-to-refresh" (for instance each sample title also uses "Pull-To-Refresh") but in the XCG the name was slightly changed to "PullToRefresh" as all of the other controls are written in PascalCase style. 

For consistency in the XCG though, I think that you raise a good point. We could change every "Pull-to-refresh" to "PullToRefresh". What do you think?

[the remark this answers is not in the document]
Comment (chingucoding) [reply XamlControlsGallery/DataModel/ControlInfoData.json:462]: Huh that's interesting that the docs uses the hyphenated name exclusively. From a nit-picky point, I would say pascal case is correct, the control is called `PullToRefresh` and if one wants to use it, they will have to type out that. Of course this does not mean that this has to be the only way, given that the docs use the hyphenated version. But I think if we are using the pascal case everywhere, having one control not do that might be a bit strange. Maybe using the pascal case is better here for consistencies sake?

[the remark this answers is not in the document]
Comment (gregwoo-microsoft) [reply XamlControlsGallery/DataModel/ControlInfoData.json:462]: I agree. I'll update this branch :)

**Review by chingucoding — APPROVED** [review]
Looks good to me, especially the updated description!

**Pull request merged**